ddtrace/appsec/_iast/taint_sinks/command_injection.py

```diff
@@ -12,6 +12,7 @@
 from ddtrace.settings.asm import config as asm_config
 
 from .._logs import iast_error
+from .._logs import iast_propagation_sink_point_debug_log
 from .._overhead_control_engine import oce
 from ._base import VulnerabilityBase
 
```

Comment:
absolute import is better than relative one, especially for static analysis

Reply:
Can I update all imports in `ddtrace/appsec/_iast/taint_sinks/*` to absolute import in a different PR :+1: